velox/functions/sparksql/tests/ComparisonsTest.cpp

```diff
@@ -409,8 +557,6 @@ TEST_F(ComparisonsTest, dateTypes) {
 
 TEST_F(ComparisonsTest, notSupportedTypes) {
   const auto candidataFuncs = {
-      "equalnullsafe",
-      "equalto",
       "lessthan",
       "lessthanorequal",
       "greaterthan",
```

Comment:
nit: @zhli1142015  do you mind reformatting this into a single line like:
```
  const auto candidataFuncs = {
      "lessthan", "lessthanorequal", "greaterthan", "greaterthanorequal"};
```
This is required to fulfill linter requirements

Reply:
Done, thanks